fixtures/docstring-proc-macro/tests/bindings/test_docstring.py

```diff
@@ -36,7 +36,13 @@
 assert AssociatedErrorTest.Test2.__doc__ == "<docstring-associated-error-variant-2>"
 
 # Test objects
-assert ObjectTest.__doc__ == "<docstring-object>"
+expected_object_doc = """
```

Comment:
it wasn't immediately clear to me why this was expected. Can you please see if a test in fixtures/procmacros isn't too difficult, and there you can test the string without the language-specific indents added by languages, and also comment about exactly why the value is what it is (ie, describe in words the intent of the dedent). Ideally here you'd also explain the same (but in this case, IIUC, Python has added an indent to match the indent of the object definition in the generated .py code?)